### PR TITLE
Blogpost: Add post about enhancements on kubernetes release team

### PR DESCRIPTION
**Pull request by rhockenbury**
This PR adds a blog post about the enhancements subteam on the release team.

### CODE REVIEW

**Comment by rhockenbury** [comment]
@gracenng @Priyankasaggu11929 @JamesLaverack @cici37 @reylejano - Would appreciate any feedback on this.  Thanks.

**Comment by Priyankasaggu11929** [comment]
LGTM. I reviewed the blog post offline.

@rhockenbury, thank you so much for writing it. 🙂

**Comment by jberkus** [comment]
/hold
for editing as a document.

**Comment by rhockenbury** [comment]
Sounds good, I actually have a copy as a [Google doc](https://docs.google.com/document/d/1QbewpB81pVme_G-iFdLr0fnBDXCAWdnoDsYmafeA7wM/edit?usp=sharing).

**Comment by rhockenbury** [comment]
Thank you @nate-double-u.  All those changes looked good, and I've accepted the edits in the Google Doc.  If you wouldn't mind, can you take one more look at [my response to this comment]? (https://docs.google.com/document/d/1QbewpB81pVme_G-iFdLr0fnBDXCAWdnoDsYmafeA7wM/edit?disco=AAAAcYC6OlM). I'll update this PR once we have that settled down.

**Comment by nate-double-u** [comment]
I took a look, and agree with your change there @rhockenbury :)

**Comment by rhockenbury** [comment]
> I took a look, and agree with your change there @rhockenbury :)

Updated this PR with all the edits.  Let me know if there's anything else that needs to be taken care of.

**Comment by jberkus** [comment]
@rhockenbury no, we just need to schedule the post.

**Comment by jberkus** [comment]
@rhockenbury I want to double-check with the main blog team to make sure they don't have any edits, if they want it on the main blog.

**Comment by rhockenbury** [comment]
Sounds great.  I did intend to publish on the main blog and I have [this PR](https://github.com/kubernetes/website/pull/34946) open to eventually do that (after this one goes through).

**Comment by rhockenbury** [comment]
Thanks for the review @rolfedh.  Updated this PR with all your feedback from the Google doc.

**Comment by nate-double-u** [comment]
/lgtm
Thanks for this @rhockenbury!

File: content/en/blog/2022/2022-07-25-enhancing-kubernetes-one-kep-at-a-time/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+The enhancements subteam verifies that all PRs for an enhancement are merged into the [Kubernetes codebase](https://github.com/kubernetes/kubernetes) (k/k). During this period, the subteam reaches out to KEP authors to understand what PRs are part of the KEP, verifies that those PRs get merged, and then updates the status of the KEP. The enhancement is removed from the release if the code isn't all merged before the code freeze deadline.
+
+## How can I get involved with the release team?
+
+I'm glad you asked. The most direct way is to apply to be a [release team shadow](https://github.com/kubernetes/sig-release/blob/master/release-team/shadows.md). The shadow role is a hands-on entry-level position intended to prepare individuals for leadership positions on the release team. Many shadow roles are non-technical and do not require prior contributions to the Kubernetes codebase.
```
Comment (reylejano) [inline content/en/blog/2022/2022-07-25-enhancing-kubernetes-one-kep-at-a-time/index.md:46]: suggestion as "entry-level position" conveys employment even though the everyone on the release team are volunteers

```suggestion
I'm glad you asked. The most direct way is to apply to be a [release team shadow](https://github.com/kubernetes/sig-release/blob/master/release-team/shadows.md). The shadow role is a hands-on apprenticeship intended to prepare individuals for leadership positions on the release team. Many shadow roles are non-technical and do not require prior contributions to the Kubernetes codebase.
```

**Comment by reylejano** [comment]
one nit but overall looks good to me
cc: SIG Release tech lead @jeremyrickard , current and past enhancements leads @Priyankasaggu11929 @salaxander @JamesLaverack @annajung

File: content/en/blog/2022/2022-07-25-enhancing-kubernetes-one-kep-at-a-time/index.md (hunk 1 above)
Comment (rhockenbury) [reply content/en/blog/2022/2022-07-25-enhancing-kubernetes-one-kep-at-a-time/index.md:46]: Ah, good callout.  Will update.

**Comment by reylejano** [comment]
Thank you for the update
/lgtm

**Comment by rhockenbury** [comment]
@jberkus - I've addressed all feedback so far.  Let me know if it's possible to get this post scheduled.  Thanks!

**Comment by rhockenbury** [comment]
Gentle bump.  Hoping to get this in ahead of the announcement for release team shadows which will be happening in a few weeks.  @jberkus @jeefy @mrbobbytables

**Comment by cici37** [comment]
/lgtm
Thank you again for raising this!

**Comment by jberkus** [comment]
@sftim what are we waiting for on this blog post, if anything?

File: content/en/blog/2022/2022-07-25-enhancing-kubernetes-one-kep-at-a-time/index.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+layout: blog
+title: "Enhancing Kubernetes one KEP at a Time"
+date: 2022-07-25
```
Comment (sftim) [inline content/en/blog/2022/2022-07-25-enhancing-kubernetes-one-kep-at-a-time/index.md:4]: I'll take the liberty of bumping the publication date.
@rhockenbury, for this blog we usually _omit_ the publication date from the filename (it's not a strong convention). For a single-file article we usually wouldn't have that as a single Markdown file too - no directory.

We can merge this as-is and fix those details later as they have no impact on the published article.


```suggestion
date: 2022-08-11
```

**Comment by sftim** [comment]
We need to future-date this (it looks wrong to add a new article that published last month), but otherwise I think we're good to go.

@rhockenbury if you have time, please also locally squash this to 1 commit and force-push. No worries if you're not already used to doing that; if you let us know, we can get Prow & GitHub to do something similar.

**Comment by sftim** [comment]
/hold cancel

editing done

**Comment by sftim** [comment]
but:

/hold
We need to update the publication date for the article

**Comment by rhockenbury** [comment]
@sftim - Please take another look.  I've addressed your comments.

**Comment by sftim** [comment]
/lgtm
/approve

Thanks @rhockenbury

**Comment by sftim** [comment]
Maybe now?
/approve

**Pull request merged**